src/config/syscheck-config.c

```diff
@@ -1220,10 +1222,23 @@ static void parse_synchronization(syscheck_config * syscheck, XML_NODE node) {
                 syscheck->enable_registry_synchronization = r;
             }
 #endif
+        } else if (strcmp(node[i]->element, xml_min_sync_interval) == 0) {
+            uint32_t interval = strtoul(node[i]->content, NULL, 10);
+
+            if (errno == ERANGE) {
+                mwarn(XML_VALUEERR, node[i]->element, node[i]->content);
+            } else {
+                syscheck->min_sync_interval = interval;
+            }
         } else {
             mwarn(XML_INVELEM, node[i]->element);
         }
     }
+
+    if (syscheck->min_sync_interval >= syscheck->sync_interval) {
+        mwarn("Synchronization min_interval is higher or equal than synchronization interval. " \
+              "Synchronization may be ommitted.");
```

Comment:
I'd modify this message because not every sync will be omitted, only those close to the latest executed. Why not something like that?
```
mwarn("Sync min_interval value higher than interval. Some synchronization turns can be skipped.");
```